apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -0,0 +1,89 @@
+/*global ServiceMixin */
+'use strict';
+
+importScripts('../service_mixin.js');
+
+(function() {
+  var service;
+
+  var ConversationService = {
+    SERVICE_NAME: 'conversation-service',
+
+    STREAMS: ['getAllConversations', 'getMessagesForConversation'],
+    METHODS: [
+      'deleteConversations', 'deleteMessages', 'markConversationsAs',
+      'getConversationMetadata', 'getMessage', 'findConversationFromNumber'
+    ],
+
+    /**
+     * Initialize our service using the Service mixin's initService.
+     */
+    init() {
+      service = this.initService();
+    },
+
+    /**
+     * Stream that returns everything needed to display conversations in the
+     * inbox view.
+     * @returns {Stream.<ConversationSummary>}
```

Comment:
nit: JSDoc is not full/correct here, we accept and return stream? Looking at @steveck-chung PR I see we can only accept stream.